src/containers/RepositoriesContainer/RepositoriesDucks.js

```diff
@@ -33,6 +34,12 @@ const REDUCERS = {
     ...state,
     isLoading: false,
     error: action.error
+  }),
+  /*When SEARCH action is triggered --filter the repos by the repo name
+    that starts with the inputValue payload.*/
+  [SEARCH]: (state, action) => ({
+    ...state,
+    repos: state.repos.filter((repo) => repo.name.startsWith(action.payload))
```

Comment:
nice usage of `startsWith` here, however IE users will suffer.
https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/String/startsWith

Can we substitute it with lodash?